src/time_evolution/mcsolve.jl

```diff
@@ -162,7 +162,7 @@ If the environmental measurements register a quantum jump, the wave function und
 - The states will be saved depend on the keyword argument `saveat` in `kwargs`.
 - If `e_ops` is specified, the default value of `saveat=[tlist[end]]` (only save the final state), otherwise, `saveat=tlist` (saving the states corresponding to `tlist`). You can also specify `e_ops` and `saveat` separately.
 - The default tolerances in `kwargs` are given as `reltol=1e-6` and `abstol=1e-8`.
-- For more details about `alg` and extra `kwargs`, please refer to [`DifferentialEquations.jl`](https://diffeq.sciml.ai/stable/)
+- For more details about `alg` and extra `kwargs`, please refer to [`OrdinaryDiffEq.jl`](https://docs.sciml.ai/OrdinaryDiffEq/stable/)
```

Comment:
We should be more precise. For the `kwargs` we can refer to [this link](https://docs.sciml.ai/DiffEqDocs/stable/basics/common_solver_opts/), while for the `alg` we can refer to [this link](https://docs.sciml.ai/DiffEqDocs/stable/solvers/ode_solve/)

Reply:
done